subprojects/platform-play/src/main/java/org/gradle/play/internal/twirl/TwirlCompilerAdapterV10X.java

```diff
@@ -33,32 +34,42 @@
 class TwirlCompilerAdapterV10X extends VersionedTwirlCompilerAdapter {
     private static final Iterable<String> SHARED_PACKAGES = Arrays.asList("play.twirl.compiler", "scala.io"); //scala.io is for Codec which is a parameter to twirl
 
-    // Based on https://github.com/playframework/playframework/blob/2.4.0/framework/src/build-link/src/main/java/play/TemplateImports.java
-    private static final Collection<String> DEFAULT_JAVA_IMPORTS = Arrays.asList(
-        "models._",
-        "controllers._",
-        "play.api.templates.PlayMagic._",
-        "java.lang._",
-        "java.util._",
-        "scala.collection.JavaConversions._",
-        "scala.collection.JavaConverters._",
-        "play.api.i18n._",
-        "play.core.j.PlayMagicForJava._",
-        "play.mvc._",
-        "play.data._",
-        "play.api.data.Field",
-        "play.mvc.Http.Context.Implicit._");
-
-    private static final Collection<String> DEFAULT_SCALA_IMPORTS = Arrays.asList(
-        "models._",
-        "controllers._",
-        "play.api.templates.PlayMagic._",
-        "play.api.i18n._",
-        "play.api.mvc._",
-        "play.api.data._");
-
-    private final String scalaVersion;
-    private final String twirlVersion;
+    // Default imports are based on:
```

Comment:
Literally, copy and paste the code from Play repository here so that we can avoid missing any imports.